src/common/schema/application.js

```diff
@@ -54,10 +54,21 @@ module.exports = {
         'zipcode',
       ]
     },
-    date: {
+    requiredDate: {
       format: 'date',
       type: 'string'
     },
+    optionalDate: {
```

Comment:
Is this needed anymore if we know to remove the field at the client side before it's sent to the server?